app/scripts/metamask-controller.js

```diff
@@ -3920,8 +3906,6 @@ export default class MetamaskController extends EventEmitter {
     ///: END:ONLY_INCLUDE_IN
 
     // filter and subscription polyfills
```

Comment:
This comment was regarding the removed lines - it should also be removed.
```suggestion
```

Reply:
Removed here: [e51455a](https://github.com/MetaMask/metamask-extension/pull/16693/commits/e51455a36612ef7a1feb40880863e74c77b0b7f5)